doc/function_reference.md

```diff
@@ -1,5 +1,191 @@
-# Function reference
+# FunctionReference
 
-You are reading a draft of the next documentation and it's in continuous update so
-if you don't find what you need please refer to:
-[C++ wrapper classes for the ABI-stable C APIs for Node.js](https://nodejs.github.io/node-addon-api/)
+`FunctionReference` is a subclass of [Reference](reference.md), and is equivalent to
```

Comment:
Here too, and in all cases where we use class names in the documentation, we should add the name space explicitly – that is, `Napi::FunctionReference` and `Napi::Reference` in this case. Additionally, "Reference" should be in backquotes.

Reply:
@NickNaso the word "Reference" inside the square bracket also needs backticks.